app/scripts/controllers/apis/detail.js

```diff
@@ -266,19 +267,42 @@ angular.module('apiExplorerApp').controller('ApisDetailCtrl', function($rootScop
                     if ($scope.api.type === "swagger") {
                         // Load swagger's JSON definition to read the default "preferences"
                         if ($scope.api.url) {
-                            $scope.loading += 1;
-                            var apiUrl = (ABSOLUTE_URL_RE.test($scope.api.url) ? "" : $rootScope.settings.currentPath) + $scope.api.url;
 
-                            // FIXME this is a redundant fetch of the swagger.json file (redundant with search as well as with
-                            // the swagger html page fetching it as well. Figure out a way to get it once.
-                            $http.get(apiUrl).then(function (response) {
+                            var apiUrl = $scope.api.url;
+                            // if (ABSOLUTE_URL_RE.test($scope.api.url)) {
```

Comment:
Do you still need the code that is commented out here? If not, delete them

Reply:
I am suspicious that I might need it which is why I left it in there.  for vRA and AirWatch it doesn't seem to be needed, but I am not sure about general platypus.